src/services/formatting/formatting.ts

```diff
@@ -726,6 +735,7 @@ namespace ts.formatting {
 
                 let listDynamicIndentation = parentDynamicIndentation;
                 let startLine = parentStartLine;
+                let indentationOnListStartToken = parentDynamicIndentation.getIndentation();

```

Comment:
It looks like the value written here is never used. It's overwritten in both `if` and `else` branches before its sole use.